3IPs/3ip-2.md

```diff
@@ -9,92 +9,68 @@ created: 2018-11-20
+This first version of threads will allow anyone to post, without any ability to remove posts. This is achieved by using `write: ['*']` when creating the store in orbitdb.
 
-OrbitDB is [working on adding](https://github.com/orbitdb/orbit-db/pull/495) support for access controllers. When they are released we can use them for write access to threads. There is not really any benefit in changing to this until they also add the ability to remove users.
+OrbitDB is [working on adding](https://github.com/orbitdb/orbit-db/pull/495) support for access controllers. When they are released we can use them for write access to threads, as well as moderation, and potentially other features as well.
 
-## Encryption
-
-We simply use `s` to symmetrically encrypt the entries in the thread if it's a private thread.
-
-## Thread subscriptions
-
-Users can subscribe to threads. This means that they will remember the thread next time they go online, and they can ask the network for updates. When a user subscribes to a thread it simply adds the thread-id and name to the rootStore of the users 3Box. When the user unsubscribes the thread entry is removed from the rootStore.
-
-## Threads in spaces?
-
-Is this something we want to enable? Maybe as a second iteration, it adds a lot of complexity.
+## Thread creation
+Threads are created inside of [Spaces](./3ip-1.md). A thread has a `name` which could be any string. For example the name could be the address of a contract if you want to create a thread for commenting on that contract. This also means that two different spaces could have their own threads with the same name. When a thread is created it gets the name `3box.thread.<space-name>.<thread-name>` for the orbitdb logstore.
 
 ## API
 
-**List threads the user is subscribed to:**
+All of the calls below assume that you have opened a space:
 ```js
-const threadNames = box.threads.list()
-console.log(threadNames)
-> ["3box-chat", "Bounties-issue432"]
+const space = box.openSpace('aGoodSpace')
 ```
 
-**Create a new thread:**
-
-The `open` parameter is a boolean that specifies if there is write permissions to the thread or not. Open threads anyone can post to, but there is no spam protection. Note that new threads are not automatically subscribed to.
+**List threads the user has interacted with**
```

Comment:
Should this be: "List Threads a User is Following/Subscribed to"  ?

Reply:
It could be, do you think we should add a method for unsubscribing?

Reply:
I think it couldn't hurt, and could allow someone to be very efficient in the threads they follow. Makes logical sense.